src/prefect/new_flow_engine.py

```diff
@@ -50,6 +50,7 @@ class FlowRunEngine(Generic[P, R]):
     flow: Flow[P, Coroutine[Any, Any, R]]
     parameters: Optional[Dict[str, Any]] = None
     flow_run: Optional[FlowRun] = None
+    is_subflow: bool = False
```

Comment:
This doesn't appear to be used?

Reply:
Ah yeah in an earlier version I thought I might have to track that. Removing.